.github/workflows/slack-deployment-message.yml

```diff
@@ -76,9 +76,6 @@ jobs:
 
           echo "build commit_pr_number"
           commit_pr_number=$(echo $deployment_info | jq -r ".meta.githubPrId")
-          if [ "${commit_pr_number}" == "null" ]; then
```

Comment:
doesn't this stay a string `"null"` in this case now though, should we set it to `null`/`undefined` instead of a string in this case?

Reply:
Good catch. I've updated the code as per review.